tests/payment/formhelper.js

```diff
@@ -0,0 +1,24 @@
+const querystring = require('querystring');
+
+module.exports.urlgenerator = function (version, first, last, email, phone, pswd, pswd_verify, secret_answer, secret_q) {
```

Comment:
a code smell called "Too Many Parameters", you can refer to this site: https://refactoring.guru/smells/long-parameter-list

I would recommend to directly pass an object, and then try to make a default value. you can read more in this article: https://itnext.io/effective-function-parameters-in-javascript-1c4f2f4602ee